RELEASING.md

```diff
@@ -0,0 +1,90 @@
+# Releasing DART from Open Robotics fork
+
+Ubuntu Launchpad PPA (Personal Package Archive) system is being used to release
+the Open Robotics fork of DART. The debian metadata is being hosted the same
+development branch `azeey/friction_per_shape_more_params`.
+
+The procedure consists in two steps:
+
+  1. Produce a local tarball of the sources and upload the tarball to Ubuntu's
+     servers to built binaries
+  1. Imported binaries from Ubuntu PPA into packages.osrfoundation.org
+
+## Prerequisites
+
+### Credentials
+
+  * For being able to upload to Open Robotics Ubuntu PPA:
+     * Account in [Ubuntu One](https://login.ubuntu.com/)
+     * Access to https://launchpad.net/~openrobotics
+       * New members can apply after login in launchpad in that same page
+
+### Step 0: System Configurations
+
+  * Enviroment variables to use as gbp configurations for changelogs
+    (optionally in .bashrc):
+    ```
+    export DEBEMAIL="user@openrobotics.org
+    export DEBFULLNAME="Your name"
+    ```
+
+  * Software installation: [debian git-buildpackage
+    tool](https://wiki.debian.org/PackagingWithGit) and dput (to upload
+    packages to PPA)
+    ```
+    apt-get install -y git-buildpackage dput
+    ```
+
+## Step 1: Releasing into PPA
+
+The first step will create a source package from DART git checkout in the local
+system and upload it to Open Robotics PPA.
+
+**One Ubuntu distribution needs to be released at a time**. The information
+about target distribution (i.e Focal) goes only in the Changelog entry. To
+release multiple distribution, repeat this step 1 changing the changelog entry
+and uploading the new source package to the PPA.
+
+A copy of [this fork](https://github.com/ignition-forks/dart) is required to be in the system.
+
+```
+cd dart
+git checkout azeey/friction_per_shape_more_params
```

Comment:
We're going to update this branch to something more official sounding.

Reply:
509055f done.